app/src/main/java/org/sopt/at/presentation/auth/signup/SignUpActivity.kt

```diff
@@ -89,23 +86,25 @@ class SignupActivity : ComponentActivity() {
         }
     }
 
-    private fun sendUserInfo(id: String, pwd: String) {
-        val intent = Intent(this, LoginActivity::class.java)
-            .putExtra(SIGNUP_USER_INFO_KEY, LoginUser(id, pwd))
-        setResult(RESULT_OK, intent)
-    }
-
-    companion object {
-        // 정규식 패턴
-        const val ID_PATTERN = "^(?=.*[a-z]+)(?=.*\\d*)[a-z\\d]{6,12}$"
-        const val PWD_PATTERN = "^.*(?=^.{8,15}$)(?=.*\\d)(?=.*[a-zA-Z])(?=.*[~!@#$%^&*]).*$"
+    private fun setBackPressedEvent() {
+        // 안드로이드 기본 뒤로가기 이벤트 정의
+        onBackPressedDispatcher.addCallback(this, object: OnBackPressedCallback(true) {
+            override fun handleOnBackPressed() {
```

Comment:
지금은 액티비티에서 회원가입 페이지를 관리하기 위해 callback메소드를 쓰고 있잖아요.
나중에 액티비티를 하나로 두어 Composable에서 백버튼 callback을 관리하려면 BackHandler 를 사용하면됩니다.
```suggestion
            @Composable
            fun BackHandler(enabled: Boolean = true, onBack: () -> Unit): Unit
```

Reply:
오 감사합니다 참고해서 얼른 SAA 패턴으로 고쳐보겠습니닷!